core/arch/arm/kernel/unwind_arm64.c

```diff
@@ -59,6 +92,9 @@ bool unwind_stack_arm64(struct unwind_state_arm64 *frame,
 	copy_in_reg(&frame->fp, fp);
 	/* LR (X30) */
 	copy_in_reg(&frame->pc, fp + 8);
+#ifdef CFG_CORE_SYSCALL_FTRACE
+	ftrace_core_map_lr(&frame->pc);
+#endif
```

Comment:
Minor: avoid this `#ifdef` by providing a stub for this function above

Reply:
Okay.